# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../linux-ip30/hotfix-gcc.patch
# Copyright (C) 2017 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- linux-2.6.19/include/linux/compiler.h.vanilla	2017-12-30 23:02:29.407841835 +0000
+++ linux-2.6.19/include/linux/compiler.h	2017-12-30 23:03:58.927838558 +0000
@@ -37,7 +37,7 @@
 #ifdef __KERNEL__
 
 #if __GNUC__ > 4
-#error no compiler-gcc.h file for this gcc version
+# include <linux/compiler-gcc4.h>
 #elif __GNUC__ == 4
 # include <linux/compiler-gcc4.h>
 #elif __GNUC__ == 3
